tests/syntax/test_equals.py

```diff
@@ -237,4 +237,51 @@ def test_differ_with_spans(self):
         ]
 
         for a, b in messages:
-            self.assertFalse(a.equals(b, with_spans=True))
+            self.assertFalse(a.equals(b, ignored_fields=None))
+
+
+class TestIgnoredFields(unittest.TestCase):
+    def setUp(self):
+        self.parser = FluentParser()
+
+    def parse_ftl_entry(self, string):
+        return self.parser.parse_entry(dedent_ftl(string))
+
+    def test_ignore_value(self):
+        strings = [
+            ("foo = Foo", "foo = Bar"),
+        ]
+
+        messages = [
+            (self.parse_ftl_entry(a), self.parse_ftl_entry(b))
+            for a, b in strings
+        ]
+
+        for a, b in messages:
+            self.assertTrue(a.equals(b, ignored_fields=['value']))
+
+    def test_ignore_value_span(self):
+        strings = [
+            ("foo = Foo", "foo = Foobar"),
+        ]
+
+        messages = [
+            (self.parse_ftl_entry(a), self.parse_ftl_entry(b))
+            for a, b in strings
+        ]
+
+        for a, b in messages:
+            self.assertTrue(a.equals(b, ignored_fields=['span', 'value']))
+
+    def test_ignore_value_not_span(self):
+        strings = [
+            ("foo = Foo", "foo = Foobar"),
+        ]
+
+        messages = [
+            (self.parse_ftl_entry(a), self.parse_ftl_entry(b))
+            for a, b in strings
+        ]
+
+        for a, b in messages:
+            self.assertFalse(a.equals(b, ignored_fields=['value']))
```

Comment:
As we're adding this in support of comments, can you add some tests here for that in particular so that we don't regress that?